lib/player.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'ui.rb'
+require_relative 'board.rb'
+
+class Player
+  attr_accessor :sign
+  def initialize
+    self.sign = nil
+  end
+
+  def assign_sign(sign)
```

Comment:
I assume `sign` in this case comes from user input right? Perhaps it will be better to simply make our `initialize` method handle this instead of having a separate method:

```
def initialize(sign)
  @sign = sign
end
```